ksql-parser/src/main/java/io/confluent/ksql/util/WithClauseUtil.java

```diff
@@ -1,50 +0,0 @@
-/*
- * Copyright 2019 Confluent Inc.
- *
- * Licensed under the Confluent Community License (the "License"; you may not use
- * this file except in compliance with the License. You may obtain a copy of the
- * License at
- *
- * http://www.confluent.io/confluent-community-license
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations under the License.
- */
-
-package io.confluent.ksql.util;
-
-import org.apache.commons.lang3.StringUtils;
-
-public final class WithClauseUtil {
```

Comment:
Removed the use of this as it was somewhat misleading:

1.  It was only called when processing property overrides, i.e. those passed via the `SET` command. It was never used to process with-clause properties as the name suggests.
1. It's been written to handle stripping quotes of the values, which isn't necessary for overrides
1. The error messages thrown are all about `WITH clause`, which is misleading to users.

Hence... it's gone!